pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-dataweave.adoc

```diff
@@ -0,0 +1,216 @@
++
+[NOTE]
+====
+Call only the methods for the bindings defined in the schema definition. For example, if the `attributes` binding is not used, don't call the `bind_attributes` method.
+
+To learn more about defining DataWeave expressions, see xref:policies-pdk-create-schema-definition.adoc#dataweave-expressions[DataWeave Expressions].
+====
+
+. After you bind all expression variables, obtain the result by calling the `eval` method.
+
+[[example-evaluator-code]]
+== Example Evaluator Code
+
+For example, to evaluate the DataWeave expression in the following policy schema definition:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Extension
+metadata:
+ labels:
+   title: jwt-auth
+   category: Custom
+spec:
+ extends:
+   - name: extension-definition
+     namespace: default
+ properties:
+   tokenExtractor:
+     type: string
+     format: dataweave
+     default: "#[vars.myVar]"
+     bindings:
+       payload:
+         mimeType: text
+       attributes: true
+       authentication: true
+       vars:
+         - myVar
+ required:
+   - tokenExtractor
+
+----
+
+Use the following Rust snippet:
+
+[source,rust]
+----
+// Copyright 2023 Salesforce, Inc. All rights reserved.
+mod generated;
+
+
+use anyhow::Result;
+
+
+use crate::generated::config::Config;
+use pdk::hl::authentication::{Authentication, AuthenticationHandler};
+use pdk::logger::info;
+use pdk::hl::*;
+use pdk::script::{Evaluator, HandlerAttributesBinding, Value};
+
+
+async fn request_filter(
+   state: RequestState,
+   stream: StreamProperties,
+   auth: Authentication,
+   mut evaluator: Evaluator<'_>,
+) {
+   evaluator.bind_vars("myVar", "myVal");
+
+   evaluator.bind_authentication(&auth.authentication());
+
+   let state = state.into_headers_state().await;
+   evaluator.bind_attributes(&HandlerAttributesBinding::new(state.handler(), &stream));
+
+   let state = state.into_body_state().await;
+   evaluator.bind_payload(&state);
+
+   if let Ok(value) = evaluator.eval() {
+       match value {
+           Value::Null => info!("value was null!"),
+           Value::Bool(val) => info!("value was Bool: {val}"),
+           Value::Number(val) => info!("value was Number: {val}"),
+           Value::String(val) => info!("value was String: {val}"),
+           Value::Array(val) => info!("value was Array: {val:?}"),
+           Value::Object(val) => info!("value was Object: {val:?}"),
+       }
+   }
+}
+
+
+#[entrypoint]
+async fn configure(launcher: Launcher, Configuration(bytes): Configuration) -> Result<()> {
+   let config: Config = serde_json::from_slice(&bytes)?;
+
+
+   launcher
+       .launch(on_request(|request, stream, auth| {
+           request_filter(request, stream, auth, config.token_extractor.evaluator())
+       }))
+       .await?;
+   Ok(())
+}
+----
+
+NOTE: This policy uses the `#[attributes.remoteAddress]` variable, which is useful for rate limiting.
+
+== DefaultBindings
```

Comment:
Usually in tutorials, "==" level section titles are imperative tasks, something like:
"Simplify Using the `DefaultBindings` Object"